human_robot_collaboration/src/baxter_display.cpp

```diff
@@ -49,6 +54,8 @@ class BaxterDisplay
     cv::Scalar green;
     cv::Scalar  blue;
 
+    cv::Mat icon; 
```

Comment:
Maybe we don't need a variable for the icon. Just create it locally in the function so that it will be destroyed when out of scope.